components/blitz/resources/omero/System.ice

```diff
@@ -33,6 +30,9 @@ module omero {
         ["java:type:java.util.ArrayList<Integer>:java.util.List<Integer>"]
             sequence<int> IntList;
 
+        ["java:type:java.util.ArrayList<String>:java.util.List<String>"]
+            sequence<string> StringSet;
```

Comment:
`omero/Collections.ice:            sequence<string> StringSet;` is inappropriate here?